third-party/arcanum-worlds/odyssey-of-the-dragonlords/aw-ootd-weapons.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<elements>
+	<info>
+		<author url="https://www.arcanumworlds.com/">Arcanum Worlds</author>
+		<update version="0.0.1">
+			<file name="aw-ootd-weapons.xml" url="https://raw.githubusercontent.com/aurorabuilder/elements/master/third-party/arcanum-worlds/odyssey-of-the-dragonlords/aw-ootd-weapons.xml" />
+		</update>
+	</info>
+
+	<!-- martial -->
+	<element name="Chakram" type="Weapon" source="Odyssey of the Dragonlords" id="ID_AW_OOTD_PHB_WEAPON_CHAKRAM">
+		<supports>ID_INTERNAL_WEAPON_CATEGORY_MARTIAL_MELEE, ID_INTERNAL_DAMAGE_TYPE_SLASHING, ID_INTERNAL_WEAPON_PROPERTY_LIGHT, ID_INTERNAL_WEAPON_PROPERTY_THROWN, ID_INTERNAL_WEAPON_PROPERTY_FINESSE, ID_INTERNAL_WEAPON_PROPERTY_SPECIAL_CHAKRAM</supports>
+		<description>
+			<p></p>
+            <div element="ID_AW_OOTD_WEAPON_PROPERTY_SPECIAL_CHAKRAM" />
+		</description>
+		<setters>
+			<set name="category">Weapons</set>
+			<set name="cost" currency="gp">10</set>
+			<set name="weight" lb="2">2 lb.</set>
+			<set name="slot">onehand</set>
+            <set name="range">60/120</set>
+			<set name="damage" type="slashing">1d6</set>
+			<set name="proficiency">ID_PROFICIENCY_WEAPON_PROFICIENCY_CHAKRAM</set>
+		</setters>
+	</element>
```

Comment:
when adding new weapons you'll also have to create a proficiency element for it and include the proficiency in the correct 'parent' proficiency (e.g. simple/martial)

(example: [Double-Bladed Scimitar](https://github.com/aurorabuilder/elements/blob/master/supplements/eberron-rising-from-the-last-war/items.xml#L674) from Eberron: Rising from the Last War)